src/util.cpp

```diff
@@ -418,6 +418,7 @@ bool SoftSetArg(const std::string& strArg, const std::string& strValue)
     if (mapArgs.count(strArg))
         return false;
     mapArgs[strArg] = strValue;
+    _mapMultiArgs[strArg].push_back(strValue);
```

Comment:
It can't get here unless the list is currently empty. It's used to initialise `-wallet`, which is accessed as a list only now.